tests/test_base_resources.py

```diff
@@ -460,6 +460,14 @@ def test_download_ylt(self, reqmock):
         f = FooView(id='abc123')
         assert f.download_ylt() == b'ylt-data'
 
+    def test_unknown_resource_equality(self, reqmock):
+        reqmock.get('https://api/foo/abc123', status_code=200,
+                    text='{"id": "abc123", "foo": "bar"}')
```

Comment:
I think I avoided implementing this because it was so costly to resolve both objects and compare them, and we had problems with this in the last version of the Python client. How necessary is this?